src/main/java/it/geosolutions/httpproxy/HTTPProxy.java

```diff
@@ -369,7 +370,8 @@ public void doPost(HttpServletRequest httpServletRequest,
                 } else if ("password".equals(key)) {
                     password = value;
                 } else if ("url".equals(key)) {
-                    url = new URL(value);
+                    url = Utils.buildURL(value);
+                    LOGGER.info("URL port is set to : " + url.getPort());
```

Comment:
Could you move the log inside the buildURL method to avoid repetition and execute it only if debug level is enabled? Having it at level info is too much.

Reply:
done